api/SVGStyleElement.json

```diff
@@ -43,6 +43,40 @@
           "deprecated": false
         }
       },
+      "disabled": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGStyleElement/disabled",
+          "spec_url": "https://svgwg.org/svg2-draft/styling.html#__svg__SVGStyleElement__disabled",
+          "support": {
+            "chrome": {
+              "version_added": "102"
```

Comment:
This is based on this bug https://bugs.chromium.org/p/chromium/issues/detail?id=695984 - the last commit mentioned first appears in Chrome 102.